salt/kubernetes-master/init.sls

```diff
@@ -100,14 +100,19 @@ kube-controller-manager:
     - source:      salt://kubernetes-master/addons/namespace.yaml.jinja
     - template:    jinja
 
-/root/skydns-rc.yaml:
+/root/kubedns-cm.yaml:
```

Comment:
Some other _addons_ packages install this kind of YAMLs in `/etc/kubernetes/addons` ([see this](https://build.opensuse.org/package/view_file/Virtualization:containers/kubernetes-addons-cilium/kubernetes-addons-cilium.spec) spec file).